lib/improver/spotdata/tests/test_neighbour_finding.py

```diff
@@ -0,0 +1,753 @@
+# -*- coding: utf-8 -*-
```

Comment:
Should this file still be here? The other tests have moved to lib/improver/tests/spotdata/.